exercise/submission_models.py

```diff
@@ -124,6 +124,81 @@ def annotate_submitter_points(
             })
         )
 
+    def annotate_submitter_points_best(
+            self,
+            field_name: str = 'total',
+            revealed_ids: Iterable[int] = None,
+            include_unofficial: bool = False,
+            ) -> 'SubmissionQuerySet':
+        """
+        Annotates the total points earned by the submitter in the exercise to
```

Comment:
Also here the comments could highlight that this is a quick fix for an immediate need that should be improved later.